static/js/libs/arrays.js

```diff
@@ -0,0 +1,26 @@
+function arraysEqual(oldArray, newArray) {
```

Comment:
Just add a comment explaining what the function does (especially to mention that arrays containing same elements in different order are considered equal).

Reply:
Done